lwt/jbuild

```diff
@@ -0,0 +1,8 @@
+(library
+ ((name        conduit_lwt)
+  (public_name conduit-lwt)
+  (preprocess (pps (ppx_sexp_conv)))
+  (wrapped false)
+  (modules (resolver_lwt))
+  (libraries (conduit lwt))))
+(jbuild_version 1)
```

Comment:
@avsm it can be anywhere. I guess if one day we change the lexer then files using the new version will have to put the version first. For now it doesn't matter